Jetchat/app/src/main/java/com/example/compose/jetchat/conversation/UserInput.kt

```diff
@@ -369,6 +393,7 @@ private fun NotAvailablePopup(onDismissed: () -> Unit) {
 val KeyboardShownKey = SemanticsPropertyKey<Boolean>("KeyboardShownKey")
 var SemanticsPropertyReceiver.keyboardShownProperty by KeyboardShownKey
 
+@OptIn(ExperimentalAnimationApi::class)
```

Comment:
AnimatedContent shouldn't be experimental anymore, can you double check if you need this?

Reply:
In Compose Animation 1.4.3 (latest stable release) it's still marked as experimental.

Reply:
Yeah, 1.5.0 then, bummer.